ExecutionHelper/hardhat.config.ts

```diff
@@ -26,8 +26,8 @@ const config: HardhatUserConfig = {
       zksync: true,
     },
     inMemoryNode: {
-      url: "http://localhost:8011",
-      ethNetwork: "", // in-memory node doesn't support eth node; removing this line will cause an error
+      url: "http://127.0.0.1:8011",
+      ethNetwork: null, // in-memory node doesn't support eth node; removing this line will cause an error
```

Comment:
The typescript LSP told me to replace `null` with `undefined`. We should check wether that works.

Reply:
I think this should actually be an empty string, I'll put that.